mackerel-plugin-windows-process-stats/lib/process-stats_windows.go

```diff
@@ -0,0 +1,113 @@
+// +build windows
+
+package mpwindowsprocessstats
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"sort"
+	"strconv"
+
+	"github.com/StackExchange/wmi"
+	mp "github.com/mackerelio/go-mackerel-plugin-helper"
+	"github.com/mackerelio/golib/logging"
+)
+
+var logger = logging.GetLogger("metrics.plugin.windows-process-stats")
+
+type Win32_PerfFormattedData_PerfProc_Process struct {
+	ElapsedTime          uint64
+	Name                 string
+	IDProcess            uint32
+	PercentProcessorTime uint64
+	WorkingSet           uint64
+}
+
+type WindowsProcessStatsPlugin struct {
+	Process     string
+	MetricLabel string
+}
+
+func getProcesses(processName string) ([]Win32_PerfFormattedData_PerfProc_Process, error) {
+	var procs []Win32_PerfFormattedData_PerfProc_Process
+
+	q := wmi.CreateQuery(&procs, "WHERE Name like '"+processName+"%'")
```

Comment:
This query has bug.

`%` is like wildcard.
This is because I want to acquire all of the specific processes.
But, this query may get to unrelated processes.


Example, if I want `chrome` processes, specifying `chrome` will also get `chromium`.
```
chromium
chrome
chrome#1
chrome#2
chrome#3
```

So, I will change it.
`/"WHERE Name like 'chrome%'"/"WHERE (Name like 'chrome' OR Name like 'chrome#%')"/`